assets/tuned/daemon/tuned/plugins/plugin_scheduler.py

```diff
@@ -83,6 +84,18 @@ def __init__(self, monitor_repository, storage_factory, hardware_inventory, devi
 		self._irq_storage_key = self._storage_key(
 				command_name = "irq")
 
+	def _calc_mmap_pages(self, mmap_pages):
+		if mmap_pages is None:
+			return None
+		try:
+			mp = int(mmap_pages)
+		except ValueError:
+			return 0
+		if mp <= 0:
+			return 0
```

Comment:
no warning on parsing error?
(just asking, I have no clue where the warnings would end up :#)

Reply:
hum, forget, `0` is used then to trigger the warning!